ui/src/app/components/webcomponents/get-started-base.component.ts

```diff
@@ -399,14 +399,14 @@ export class GetStartedBaseComponent extends BaseComponent implements OnInit {
 
   private fetchDryTestCaseCount(user) {
     if (this.userCreatedAt)
-      this.dryTestPlanService.findAll("createdDate>" + moment(this.userCreatedAt).format("YYYY-MM-DD")).subscribe(
+      this.dryTestPlanService.findAll("createdDate>" + moment(this.userCreatedAt)).subscribe(
       res => this.getStartedCounts["dryTestCaseCount"] = res.content.length
```

Comment:
@nixonakash01 Is there any other network error ?

Reply:
no there is not